_layouts/quality_estimation.html

```diff
@@ -0,0 +1,124 @@
+---
+layout: default
+parent: Quality estimation
+---
+
+<h1>
+  {{ page.title }}
+</h1>
+<h2>
+  {{ page.description }}
+</h2>
+
+{{ content }}
+
+<hr/>
+
+<p>
+  {% if page.supported_languages.size > 0 %}
+   <strong>{{ page.title }}</strong> supports <a href="#language-support">{{ page.supported_languages | size }} languages</a>.
+  {% else %}
+    The <strong>{{ page.title }}</strong> supports an unknown number of languages.
+  {% endif %}
+</p>
+
+<p>
+  {% if page.customisation %}
+  <a href="/customisation">Customisation</a> is supported.
```

Comment:
This is a bit misleading, because that page refers to customization of machine translation, not quality prediction, but it's fine.